docs/src/pages/components/menus/ContextMenu.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import Typography from '@material-ui/core/Typography';
+
+export default function ContextMenu() {
+  const [state, setState] = React.useState({
+    anchorEl: null,
+    mouseX: null,
+    mouseY: null,
+  });
+  const handleClick = event => {
+    event.preventDefault();
+    setState({
+      ...state,
+      mouseX: event.clientX - 2,
+      mouseY: event.clientY - 4,
+      anchorEl: event.currentTarget,
+    });
+  };
+
+  const handleClose = () => {
+    setState({ ...state, anchorEl: null });
+  };
+  const { mouseX, mouseY, anchorEl } = state;
+  return (
+    <div aria-controls="simple-menu" aria-haspopup="true" onContextMenu={handleClick}>
```

Comment:
I would go with the first rule of aria here and don't use it. I don't think they are intended for context menus. Unless you you have some link to the WAI-ARIA spec or their authoring guidelines.

Also: There is a special cursor style for context menu which makes this easier to discover for mouse users.

Reply:
![Screenshot 2019-10-11 at 11 32 29 PM](https://user-images.githubusercontent.com/15229752/66673976-6cb03600-ec7f-11e9-959b-5e5c5d3666c3.png)


Is it this one?

Reply:
Yep.